bin/setup.sh

```diff
@@ -6,7 +6,7 @@
 set -euxo pipefail
 
 # Set up conan
-pip install conan
+conan --version > /dev/null 2>&1 || python -m pip install conan
```

Comment:
Drive-by: if you already have `conan` available on your path, don't try to install it again. Also, `python -m pip` is preferred, per my understanding.